numcosmo/lss/nc_galaxy_wl_dist.h

```diff
@@ -90,6 +93,15 @@ G_END_DECLS
 
 G_BEGIN_DECLS
 
+NCM_INLINE void
+nc_galaxy_wl_dist_m2lnP_initial_prep (NcGalaxyWLDist *gwld,NcGalaxyRedshift *gz, NcHICosmo *cosmo, NcHaloDensityProfile *dp, NcWLSurfaceMassDensity *smd, const gdouble z_cluster)
+{
+  NcGalaxyWLDistClass *gwld_class = NC_GALAXY_WL_DIST_GET_CLASS (gwld);
+  
+  if (gwld_class->m2lnP_initial_prep != NULL)
+    return NC_GALAXY_WL_DIST_GET_CLASS (gwld)->m2lnP_initial_prep (gwld, gz, cosmo, dp, smd, z_cluster);
```

Comment:
Why include return in a void function?

Reply:
I copied it from m2lnP_prep, which is also a void function. Should that be changed too?